index.html

```diff
@@ -2612,6 +2617,96 @@ <h3 id="applying">
+        </h3>
+        <p>
+          The <dfn id="member-category"><code>category</code>
+          member</dfn> describes the expected application category (or 
+          categories) to which the web application belongs. This is a hint 
+          to catalogs or stores of web applications under which category 
+          (or categories) to list this web application.
+        </p>
+        <p>
+          The <code>category</code> member is only meant as a hint to catalogs 
+          or stores listing web applications and it is expected that these 
+          will make a best effort to find an appropriate category under which 
+          to list the web application.
+        </p>
+        <p>
+          The <dfn>steps for processing the <code>category</code>
+          member</dfn> are given by the following algorithm. The algorithm
+          takes a <var>manifest</var> as an argument. This algorithm returns a
+          string or <code>undefined</code>. The string MAY include a comma 
+          delimited list of categories.
+        </p>
+        <ol>
+          <li>Let <var>value</var> be the result of calling the
+          <a>[[\GetOwnProperty]]</a> internal method of <var>manifest</var>
+          with argument "<code>category</code>".
+          </li>
+          <li>If <a>Type</a>(<var>value</var>) is not "string":
+            <ol>
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined", <a>issue
+              a developer warning</a> that the type is not supported.
+              </li>
+              <li>Return <code>undefined</code>.
+              </li>
+            </ol>
+          </li>
+          <li><a>Trim</a>(<var>value</var>) and convert to lower-case</li>
+          <li>Return <var>category</var>.
+          </li>
+        </ol>
+        <p>
+          The category string is case insensitive and converted to lower-case 
+          by following the processing algorithm. Thus, <code>sports</code>, 
+          <code>Sports</code>, <code>SPORTS</code>, and <code>SpOrTs</code> 
+          are all equivalent. Manifest authors SHOULD use lower-case.
+        </p>
+        <p>
+          The category string is a hint to catalogs and stores were the 
+          Manifest author suggests their web app should be listed. 
+          This is hint or suggestion from the Manifest author. Like 
+          search engine keywords, catalogs and stores are not required 
+          to honor this hint.</p>
+        <p>
+          If a category string has multiple comma delimited values, a 
+          consuming system MAY use one or more of these values.  
+        </p>
+        <p>
+          Manifest authors SHOULD use the below common categories:
+        </p>
+        <ul>
+          <li>books</li>
+          <li>business</li>
+          <li>education</li>
+          <li>entertainment</li>
+          <li>finance</li>
+          <li>food</li>
+          <li>games</li>
+          <li>government & politics</li>
```

Comment:
Good question @kenchris. based on an analysis of what categories are used by the Apple, Google, and Microsoft stores (below), I believe we should treat health and fitness as a single category since all three of these stores do so: 

| Apple | Google | Microsoft |
 Health & Fitness | Health & Fitness | Health + fitness |